department-of-veterans-affairs/va.gov-cms
# VACMS-5757: Remove node link report whitelist from config.

## Description
See #5757
@swirtSJW - this patch is probably going to have to remain a patch that isn't merged into the node_link_report project. I _think_ our not wanting the whitelist to be in config is unique to VA.

## Testing done
Visual

## QA steps
- Once deployed to prod, go to `admin/config/content/node_link_report` and alter the `Domains that should be exempt from processing.` field and save the form.
- Verify after next code deploy that changes remain.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
